src/sentry/event_manager.py

```diff
@@ -1061,7 +1062,10 @@ def _nodestore_save_many(jobs: Sequence[Job], app_feature: str) -> None:
                 usage_type=UsageUnit.BYTES,
             )
         job["event"].data["nodestore_insert"] = inserted_time
-        job["event"].data.save(subkeys=subkeys)
+        # Error events are nearly all expected to be read shortly after ingestion by
+        # post-processing tasks, so it's worth writing to cache.
+        force_cache_write = event_type == "error"
```

Comment:
This looks good to me, although I'd probably also put this behind a feature flag